src/store/filestore.rs

```diff
@@ -0,0 +1,209 @@
+use crate::{
+    error::{FileExists, FileHasNoParent, NoAnnotationFound, NoRegexMatch},
+    model::{from_yaml, to_yaml, Pod},
+    store::Store,
```

Comment:
For `Store` trait I'd fine relative import makes more sense rather than absolute path